app/models/order_detail.rb

```diff
@@ -46,6 +46,7 @@ def set_problem_order
   belongs_to :assigned_user, class_name: "User", foreign_key: "assigned_user_id"
   belongs_to :created_by_user, class_name: "User", foreign_key: :created_by
   belongs_to :dispute_by, class_name: "User"
+  belongs_to :price_changed_by_user, class_name: "User"
```

Comment:
per codeclimate: "Specify an `:inverse_of` option."

Reply:
I'm not seeing this in codeclimate for some reason? Confused.

Reply:
hmm. I have the codeclimate chrome extension, which marks certain things. Maybe there's a rubocop version discrepancy? Don't worry about it, then.